.expeditor/config.yml

```diff
@@ -0,0 +1,8 @@
+slack:
+  notify_channel: ops
+
+subscriptions:
+  - workload: artifact_published:stable:habitat:*
+    actions:
+      - bash:.expeditor/update_hab_release.sh
+      - built_in:update_changelog
```

Comment:
So, I'm sorry but for the confusion but `artifact_published` isn't a supported for `artifact_published`. It's only supported for `pull_request_merged`. If you want to update the changelog when the PR is merged, that'd be a `merge_action`.